setup.py

```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```

Comment:
@palewire 
The patch simply doesn't work with dj 1.7 regardless of the guard. 
Tests are fine.